test/modules/Modules.Test/TestResultCoordinator/Reports/DirectMethod/DirectMethodReportMetadataJsonConverterTest.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Modules.Test.TestResultCoordinator.Reports.DirectMethod
+{
+    using System;
+    using global::TestResultCoordinator.Reports;
+    using global::TestResultCoordinator.Reports.DirectMethod;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil;
+    using Newtonsoft.Json;
+    using Xunit;
+
+    public class DirectMethodReportMetadataJsonConverterTest
```

Comment:
is this still valid? I can't find any DirectMethodReportMetadataJsonConverter.cs file in this PR.

Reply:
Deleted, moved tests to TestReportUtilTest